app/services/transformer_loader.rb

```diff
@@ -61,13 +61,14 @@ def transform_records
       raise if fail_fast?
     end
     # Remove previous or non-canonical versions
-    process_versions(solr_docs)
+    process_records(solr_docs)
```

Comment:
Similarly this could be `consolidate_records`

Reply:
Sounds good. I may say consolidate_datasets but it's the same idea.